pkg/service/service.go

```diff
@@ -2104,3 +2100,11 @@ func (s *Service) SyncServicesOnDeviceChange(nodeAddressing types.NodeAddressing
 		}
 	}
 }
+
+// sendNotification sends an agent notification to the monitor
+func (s *Service) sendNotification(notification monitorAPI.AgentNotifyMessage) error {
+	if option.Config.DryMode {
```

Comment:
Not sure if need `DryMode` here. The monitor agent cell has an optional dependency to the events map and thus works fine in non-privileged integration tests.